static/js/components/dashboard/CourseDescription.js

```diff
@@ -0,0 +1,63 @@
+// @flow
+import React from 'react';
+import moment from 'moment';
+import IconButton from 'react-mdl/lib/IconButton';
+
+import { Course, CourseRun } from '../../flow/programTypes';
+import {
+  STATUS_NOT_OFFERED,
+  STATUS_PASSED,
+  STATUS_NOT_PASSED,
+  STATUS_ENROLLED_NOT_VERIFIED,
+  STATUS_VERIFIED_NOT_COMPLETED,
+  DASHBOARD_FORMAT,
+} from '../../constants';
+
+export default class CourseDescription extends React.Component {
+  props: {
+    course: Course,
+    now: Object,
```

Comment:
`now` is a momentjs object right? We have flow type annotations installed for moment.